src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -173,7 +172,7 @@ void tabulate() {
           try {
             newWinnerSet = runTabulationForConfig(config, castVoteRecords);
           } catch (TabulationCancelledException exception) {
-            Logger.severe("Tabulation was cancelled by the user!");
+            Logger.severe(exception.getMessage());
```

Comment:
Here and below: I think you might be better off just logging `exception`, since that should include the message and additional useful info as well.

Reply:
I don't think we want other info; the purpose of this exception class is to communicate a specific message to the user in the output.